### PR TITLE
6965 complete a leapp module

### PR DESCRIPTION
**Pull request by markmckinnon**

### CODE REVIEW

**Comment by kellykelly3** [comment]
@markmckinnon when you get a moment could you please fix the merge conflicts?  Thanks!

**Comment by kellykelly3** [comment]
@markmckinnon Please make sure the methods all have there appropriate headers and that  the files are formatted.

**Comment by kellykelly3** [comment]
@markmckinnon Format ALeappAnalyzerIngestModule.  There is some weird spacing.  And fix merge conflicts.

**Review by kellykelly3 — CHANGES_REQUESTED** [review]

File: Core/src/org/sleuthkit/autopsy/modules/leappanalyzers/ALeappAnalyzerIngestModule.java [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+        }
+
+        ProcessResult fileProcessorResult = aLeappFileProcessor.processFiles(dataSource, moduleOutputPath, aLeappFile);
+
+        if (fileProcessorResult == ProcessResult.ERROR) {
+            return;
+        }
+    }
+
+    private void processALeappFs(Content dataSource, Case currentCase, DataSourceIngestModuleProgress statusHelper, String directoryToProcess) {
```
[the remark this answers is not in the document]
Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/modules/leappanalyzers/ALeappAnalyzerIngestModule.java:232]: Added comments.

File: Core/src/org/sleuthkit/autopsy/modules/leappanalyzers/ALeappAnalyzerIngestModule.java [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+                    || aLeappFile.getName().toLowerCase().contains(".tgz"))) {
+                aLeappFilesToProcess.add(aLeappFile);
+
+            }
+        }
+
+        return aLeappFilesToProcess;
+    }
+
+    private ProcessBuilder buildaLeappCommand(Path moduleOutputPath, String sourceFilePath, String aLeappFileSystemType) {
```
[the remark this answers is not in the document]
Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/modules/leappanalyzers/ALeappAnalyzerIngestModule.java:315]: Added comments

**Review by kellykelly3 — APPROVED** [review]

**Pull request merged**